tutorials/physics/interpolation/using_physics_interpolation.rst

```diff
@@ -0,0 +1,72 @@
+
+Choose a physics tick rate
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This is something that you may have never changed before, but when using physics interpolation, the rendering is decoupled from physics, and you can choose something that makes sense for your game.
+
+As a rough guide:
+
+.. csv-table::
+    :header: "Low tick rates (10-30)", "Medium tick rates (30-60)", "High tick rates (60+)"
+    :widths: 20, 20, 20
+    
+    "Better CPU performance","Good physics behaviour in complex scenes","Good with fast physics"
+    "Add some delay to input","Good for first person games","Good for racing games"
+    "Simple physics behaviour"
+
+.. note:: You can always change the tick rate as you develop, it is as simple as changing the project setting.
+
+Call reset_physics_interpolation when teleporting objects
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Although 99% of the time interpolation is what you want between two physics ticks, there is one situation in which it may *not* be what you want. That is when you are initially placing objects, or moving them to a new location, when you do not want a smooth motion between the two, but an instantaneous move.
+
+The solution to this is quite simple, each :ref:`Node<class_Node>` has a :ref:`Node.reset_physics_interpolation<class_Node_method_reset_physics_interpolation>` function which you can call *after* setting the position / transform. The rest is done for you automatically.
+
+Even if you forget to call this, it is not usually a problem in most situations (especially at high tick rates), and is something you can easily leave to the polishing phase of your game. The worst that will happen is seeing a streaking motion for a frame or so when you move them - you will know when you need it!
+
+.. important:: You should call ``reset_physics_interpolation()`` *after* setting the new position, rather than before, otherwise you may still see the unwanted streaking motion.
+
+Testing and Debugging Tips
+--------------------------
+
+As hinted earlier, even if you intend to run physics at 60tps, in order to thoroughly test your interpolation and get the smoothest gameplay, it is highly recommended to temporarily set the physics tick rate to a low value such as 10tps.
+
+The gameplay may not work perfectly, but it should enable you to more easily see cases where you should be calling :ref:`Node.reset_physics_interpolation<class_Node_method_reset_physics_interpolation>`, or where you should be using your own custom interpolation on e.g. a :ref:`Camera<class_Camera>`. Once you have these cases fixed, you can set the physics tick rate back to the desired setting.
+
+The other great advantage to testing at low tick rate is you can often notice other game systems that are synchronized to the physics tick and creating glitches which you may want to work around. Typical examples include setting animation blend values, which you may decide to set in ``process()`` and interpolate manually.
```

Comment:
```suggestion
Testing and debugging tips
--------------------------

Even if you intend to run physics at 60 TPS, in order to thoroughly test your interpolation and get the smoothest gameplay, it is highly recommended to temporarily set the physics tick rate to a low value such as 10 TPS.

The gameplay may not work perfectly, but it should enable you to more easily see cases where you should be calling :ref:`Node.reset_physics_interpolation<class_Node_method_reset_physics_interpolation>`, or where you should be using your own custom interpolation on e.g. a :ref:`Camera<class_Camera>`. Once you have these cases fixed, you can set the physics tick rate back to the desired setting.

The other great advantage to testing at a low tick rate is you can often notice other game systems that are synchronized to the physics tick and creating glitches which you may want to work around. Typical examples include gameplay variables that vary depending on physics tick rate (such as jump height or damage over time). Another common issue is setting animation blend values, which you may decide to set in ``_process()`` and interpolate manually.
```

Reply:
On the change at line 72, while it is true you can notice gameplay bugs that are tick rate dependent like jump height / damage over time, these are not related to physics interpolation, so I'm not 100% sure whether it should be in this section. :thinking: Whereas blend values are, because these may need to be manually interpolated.